requirements.lock

```diff
@@ -8,38 +8,38 @@
 #   with-sources: false
 
 -e file:.
-aiohttp==3.9.1
+aiohttp==3.9.3
 aiosignal==1.3.1
 annotated-types==0.6.0
 appdirs==1.4.4
 async-timeout==4.0.3
-attrs==23.1.0
-bitsandbytes==0.41.3
+attrs==23.2.0
+bitsandbytes==0.42.0
```

Comment:
See bitsandbytes 0.42.0 release notes:

> Fixed added missing scipy requirement @dulalbert https://github.com/TimDettmers/bitsandbytes/pull/525

I think we'll be able to drop it here -> https://github.com/finegrain-ai/refiners/blob/45357c55480a8e834c0ab19a7ae7c31f7e0d777b/pyproject.toml#L25-L27